List-Comprehensions.ipynb

```diff
@@ -95,8 +95,13 @@
    "cell_type": "code",
```

Comment:
<p>Typo: have experience that <strong>being</strong> too eager to make <strong>things</strong> into comprehensions is actually a bad idea</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>